ansible/roles/subjects/templates/subject_versions.fact

```diff
@@ -0,0 +1,8 @@
+[subjects]
```

Comment:
@MOZGIII is this necessary? When the facts are loaded they'll be namespaced, right? I assume we can remove this and set root level facts?

Reply:
Sounds good. Would you mind making this change?

Reply:
Sure!